QuoridorEngine/src/core/QuoridorGame.cs

```diff
@@ -300,7 +302,18 @@ private void movePlayer(QuoridorMove move)
             // Checking if another player is already located on destination coordinates
             if (white.Row == move.Row && white.Column == move.Column) throw new InvalidMoveException("This position is occupied");
             if (black.Row == move.Row && black.Column == move.Column) throw new InvalidMoveException("This position is occupied");     
+            */
 
+            QuoridorPlayer targetPlayer = getTargetPlayer(move.IsWhitePlayer);
```

Comment:
Good enough for now but maybe we could optimize this later to directly check if the move is legal or not